fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/DATA/MfdFmsDataAirport.tsx

```diff
@@ -0,0 +1,360 @@
+
+    this.isRunwayDataVisible.set(true);
+  }
+
+  private async loadAirportRunways(icao: string | null) {
+    if (icao) {
+      const airport = await loadAirport(icao);
+
+      if (airport) {
+        const runways = await loadAllRunways(airport);
+
+        const airportCoords = coordinateToString(airport.location.lat, airport.location.long, false);
+
+        const simplifiedRunways = runways.map((runway) => {
+          const length = MathUtils.round(runway.length, 10).toFixed(0).padStart(5, '\xa0');
+          const elevation = MathUtils.round(runway.thresholdLocation.alt, 10).toFixed(0).padStart(5, '\xa0');
+
+          return {
+            label: `${runway.ident.substring(4).padEnd(3, ' ')} ${length}M ${runway.lsIdent ? 'ILS' : ''}`,
+            length,
+            lsIdent: runway.lsIdent ?? '\xa0',
+            elevation,
+            coords: coordinateToString(runway.thresholdLocation.lat, runway.thresholdLocation.long, false),
+            runwayIdent: runway.ident.substring(4).padEnd(3, ' '),
+            course: runway.bearing.toFixed(0).padStart(3, '0'),
+          };
+        });
+
+        this.airportRunways.set(simplifiedRunways);
+
+        if (airport.name && airport.location) {
+          this.airportName.set(airport.name.toUpperCase());
+          this.airportCoords.set(airportCoords);
+        }
+
+        this.renderRunwayButtons();
+      }
+    }
+  }
+
+  private checkScrollButtons() {
+    const currentIndex = this.selectedRunwayIndex.get() ?? 0;
+    const totalRunways = this.airportRunways.get().length;
+
+    this.currentIndexRef.instance.innerHTML = (currentIndex + 1).toFixed(0);
+    this.totalRunwaysRef.instance.innerHTML = totalRunways.toFixed(0);
+
+    this.disabledScrollLeft.set(currentIndex === 0);
+    this.disabledScrollRight.set(currentIndex === totalRunways - 1);
+  }
+
+  protected onNewData() {}
+
+  public onAfterRender(node: VNode): void {
+    super.onAfterRender(node);
+
+    this.airportIcao.sub((icao: string | null) => {
+      if (icao && icao.length === 4) {
+        this.loadAirportRunways(icao);
+        this.isAirportInfoVisible.set(true);
+        this.isRunwayButtonListVisible.set(true);
+        this.isRunwayDataVisible.set(false);
+      } else {
+        this.isAirportInfoVisible.set(false);
+        this.isRunwayDataVisible.set(false);
+        this.isRunwayButtonListVisible.set(false);
+      }
+    });
+
+    this.selectedRunwayIndex.sub((index: number | null) => {
+      if (index === null) {
+        this.isRunwayButtonListVisible.set(true);
+        this.isRunwayDataVisible.set(false);
+      } else {
+        this.isRunwayButtonListVisible.set(false);
+        this.checkScrollButtons();
+        this.currentIndex.set(index + 1);
+        const runwayData = this.airportRunways.get()[index];
+        this.renderRunwayData(runwayData);
+      }
+    });
+
+    this.subs.push(
+      this.props.mfd.uiService.activeUri.sub((val) => {
+        if (val.extra === 'acft-status') {
```

Comment:
```suggestion
        if (val.extra === 'database') {
```
This was probably copied from DATA/STATUS, you might want to adapt the URL for the tabs

Reply:
Sorted